inventory/sample/hosts.ini

```diff
@@ -0,0 +1,11 @@
+[host_vm]
+rpi-creation    ansible_host=10.182.31.179   ansible_port=22     ansible_user=ubuntu
```

Comment:
Let's rename this to `vm` and make sure spaces between arguments are consistent.
I personnally tend to prefer using yml format for inventory files though.